hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/om/quota/QuotaStatus.java

```diff
@@ -35,7 +35,7 @@
     mixinStandardHelpOptions = true,
     versionProvider = HddsVersionProvider.class
 )
-public class QuotaStatus implements Callable<Void>  {
+public class QuotaStatus extends RepairTool {
```

Comment:
`QuotaStatus` just queries status, so the common repair steps (ask for confirmation, ensure `--dry-run` flag, etc.) do not apply.  I don't think it should extend `RepairTool` (but it can extend `AbstractSubcommand` if needed).

`info()` will not be available, but `out()` and `err()` will be after HDDS-12002.  Until then, we can leave usage of `System.(out|err)` as is.